README.md

```diff
@@ -44,4 +44,68 @@ We also automatically publish snapshots on Sonatype:
 resolvers += Resolver.sonatypeRepo("snapshots")
 
 libraryDependencies += "ch.epfl.scala" %% "collection-strawman" % "0.2.0-SNAPSHOT"
-~~~
\ No newline at end of file
+~~~
+
+## Implemented collection types
+
+- [x] `List`
+- [x] `LazyList` (aka `Stream` in 2.12)
+- [ ] `Queue`
+- [ ] `Stack`
+- [x] `ArrayOps`
+- [x] `StringOps`
+- [x] `ArrayBuffer`
+- [x] `ListBuffer`
+- [ ] `UnrolledBuffer`
+- [ ] `LinkedList`
+- [ ] `DoubleLinkedList`
+- [ ] `Range` / `NumericRange`
+- [ ] `HashMap`
+- [ ] `TreeMap`
+- [ ] `IntMap` / `LongMap` (?)
+- [ ] `ListMap`
+- [ ] `MultiMap`
+- [ ] `HashSet`
+- [ ] `ListSet`
+- [ ] `TreeSet`
+- [ ] `EqSet`
+- [ ] `BitSet`
+- [x] `View`
+
+## Implemented operations (on the relevant collection types)
+
+### Terminal operations
```

Comment:
Since the rewrite is still eager, this terminology is especially confusing since it sounds like the Java 8 Streams terminology meaning "evaluate this now".  Maybe use "Methods not returning the same collection type"?